redisdb/assets/dashboards/overview.json

```diff
@@ -1,672 +1,789 @@
+    },
+    {
+      "id": 7,
+      "definition": {
+        "type": "timeseries",
+        "requests": [
+          {
+            "q": "sum:redis.keys{$scope}",
+            "display_type": "area",
+            "style": {
+              "palette": "purple"
+            }
+          }
+        ],
+        "custom_links": [],
+        "title": "Keys",
+        "title_size": "16",
+        "title_align": "left",
+        "time": {
+          "live_span": "4h"
+        },
+        "show_legend": false,
+        "legend_size": "0"
+      },
+      "layout": {
+        "x": 124,
+        "y": 71,
+        "width": 47,
+        "height": 15
+      }
+    },
+    {
+      "id": 8,
+      "definition": {
+        "type": "query_value",
+        "requests": [
+          {
+            "q": "sum:redis.rdb.changes_since_last{$scope}",
+            "aggregator": "avg",
+            "conditional_formats": [
+              {
+                "comparator": ">",
+                "value": 1073741824,
+                "palette": "white_on_red"
+              },
+              {
+                "comparator": "<",
+                "value": 1073741824,
+                "palette": "white_on_green"
+              }
+            ]
+          }
+        ],
+        "custom_links": [],
+        "title": "Unsaved changes",
+        "title_size": "16",
+        "title_align": "left",
+        "time": {
+          "live_span": "5m"
+        },
+        "autoscale": true,
+        "precision": 2,
+        "text_align": "left"
+      },
+      "layout": {
+        "x": 83,
+        "y": 6,
+        "width": 18,
+        "height": 10
+      }
+    },
+    {
+      "id": 9,
+      "definition": {
+        "type": "timeseries",
+        "requests": [
+          {
+            "q": "diff(sum:redis.net.rejected{$scope})",
+            "display_type": "line",
+            "style": {
+              "palette": "warm"
+            }
+          }
+        ],
+        "custom_links": [],
+        "title": "Rejected connections",
+        "title_size": "16",
+        "title_align": "left",
+        "time": {
+          "live_span": "4h"
+        },
+        "show_legend": false,
+        "legend_size": "0"
+      },
+      "layout": {
+        "x": 124,
+        "y": 23,
+        "width": 47,
+        "height": 15
+      }
+    },
+    {
+      "id": 10,
+      "definition": {
+        "type": "query_value",
+        "requests": [
+          {
+            "q": "sum:redis.replication.master_link_down_since_seconds{$scope}",
+            "aggregator": "avg",
+            "conditional_formats": [
+              {
+                "comparator": ">",
+                "value": 60,
+                "palette": "white_on_red"
+              },
+              {
+                "comparator": "<",
+                "value": 60,
+                "palette": "white_on_green"
+              }
+            ]
+          }
+        ],
+        "custom_links": [],
+        "title": "Master down",
```

Comment:
What's the meaning of "Master down" ? The dashboard seems to display something strange:

![image](https://user-images.githubusercontent.com/49917914/95309303-41105d80-088b-11eb-9bd1-68723e247ef9.png)

Reply:
Not sure, it comes from the legacy dashboard. Checking it

Reply:
Removed decimals and added note